docs/design/autoware-interfaces/ad-api/features/cooperation.md

```diff
@@ -0,0 +1,91 @@
+# Cooperation
+
+## Related API
+
+- {{ link_ad_api('/api/planning/velocity_factors') }}
+- {{ link_ad_api('/api/planning/steering_factors') }}
+- {{ link_ad_api('/api/planning/cooperation/set_commands') }}
+- {{ link_ad_api('/api/planning/cooperation/set_policies') }}
+- {{ link_ad_api('/api/planning/cooperation/get_policies') }}
+
+## Description
+
+Request to cooperate (RTC) is a feature that the operator supports the decision in autonomous driving mode.
+The planning component manages each situation that requires decision as a scene.
+Autoware usually drives the vehicle using its own decisions, but the operator may prefer to make their decisions in experiments and complex situations.
+
+Each scene has an ID that doesn't change until the scene is completed or canceled.
+The operator can override the decision of the target scene using this ID.
+In practice, the user interface application can hides the specification of the ID and provides an abstracted interface to the operator.
+
+For example, in the situation in the diagram below, change lanes twice and turn left at the intersection.
+There are three scenes and each has a decision to change or keep the lane, turn left or wait.
+Here Autoware decides not to change lanes a second time due to the obstacle, so the vehicle will stop there.
```

Comment:
It might be better to include operator interaction in the example as well.

```suggestion
Here Autoware decides not to change lanes a second time due to the obstacle, so the vehicle will stop there.
However, operator could overwrite that decision through RTC function and force the lane change so that vehicle could reach to it's goal. 
```